docs/index.md

```diff
@@ -0,0 +1,12 @@
+![genie banner](https://raw.githubusercontent.com/Sage-Bionetworks/Genie/master/genie_banner.png)
+
+# AACR Project GENIE
+
+[![PyPi](https://img.shields.io/pypi/v/aacrgenie.svg?style=for-the-badge&label=PyPi&logo=PyPi)](https://pypi.org/project/aacrgenie)
+[![Docker Automated](https://img.shields.io/docker/automated/sagebionetworks/genie.svg?style=for-the-badge&logo=docker)](https://hub.docker.com/r/sagebionetworks/genie)
+[![GitHub CI](https://img.shields.io/github/actions/workflow/status/Sage-Bionetworks/Genie/ci.yml?branch=develop&style=for-the-badge&logo=github)](https://github.com/Sage-Bionetworks/Genie)
+
+
+## Introduction
+
+This repository documents code used to gather, QC, standardize, and analyze data uploaded by institutes participating in AACR's Project GENIE (Genomics, Evidence, Neoplasia, Information, Exchange).
```

Comment:
Lets link to the portal URL?

Reply:
Like move portal URL to here, and put the GH pages site to here?
<img width="357" alt="image" src="https://github.com/Sage-Bionetworks/Genie/assets/26471741/94f9d060-7b16-4468-bc9e-e8b4548fdb91">

Reply:
@rxu17 hmmm, i think its valuable to have the portal URL be linked in as many places as possible.  I did think about putting out documentation URL there.  I go back and forth, but I suppose people who land on the GENIE github probably want documentation and not the portal.

Thoughts @Chelsea-Na ?

Reply:
Got it, I added it to that intro page